### ISSUE TITLE
Cleanup error after upgrading from 2.0.0-alpha.3 to 2.0.0-alpha.4

### ISSUE THREAD

**Issue by kushdilip** [dockyard/ember-validations#327]
`ember build` is giving below errors and gets stuck indefinitely. 

```
version: 0.1.12
Cleanup error.
Object #<Babel> has no method 'cleanup'
TypeError: Object #<Babel> has no method 'cleanup'
    at cleanupTree (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/broccoli/lib/builder.js:128:19)
    at /Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/broccoli/node_modules/promise-map-series/index.js:11:14
    at $$$internal$$tryCatch (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:490:16)
    at $$$internal$$invokeCallback (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:502:17)
    at $$$internal$$publish (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:473:11)
    at Object.$$rsvp$asap$$flush [as _onImmediate] (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1581:9)
    at processImmediate [as _immediateCallback] (timers.js:354:15)
Build failed.
Invalid tree found. You must supply a path or an object with a `read` function.
Error: Invalid tree found. You must supply a path or an object with a `read` function.
    at readAndReturnNodeFor (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/broccoli/lib/builder.js:71:13)
    at RSVP.resolve.then.then.finally.readTreeRunning (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/broccoli/lib/builder.js:92:24)
    at $$$internal$$tryCatch (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:490:16)
    at $$$internal$$invokeCallback (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:502:17)
    at /Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1096:13
    at Object.$$rsvp$asap$$flush [as _onImmediate] (/Users/dilip/Documents/Code/AdminConsole-Org/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1581:9)
    at processImmediate [as _immediateCallback] (timers.js:354:15)
```

#### Output of `ember -v`
version: 0.1.12
node: 0.10.37
npm: 2.1.8

**Comment by bcardarella** [comment]
That is strange, as there aren't any build related changes from what I can see between alpha 3 to alpha 4: https://github.com/dockyard/ember-validations/compare/v2.0.0-alpha.3...master

**Comment by chrismllr** [comment]
@kushdilip Hey, had this exact same issue yesterday. was solved by updating Ember CLI. I believe one of its dependencies (RSVP maybe) got upgraded to only cooperate with a recent CLI.

I was on 0.1.8, and was solved by upgrading to 0.2.2 at least.

Hope this helps
